angular/src/docs/app/app-routing.module.ts

```diff
@@ -27,6 +28,10 @@ const routes: Routes = [
         path: 'radio',
         component: ExampleRadioKitchenSinkComponent,
       },
+      {
+        path: 'stepper',
+        component: StepperDefaultComponent,
```

Comment:
Follow format of ExampleStepperKitchenSinkComponent

Reply:
Done